### PR TITLE
Remove scalatest dependency and bump to latest versions #4

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #4
Remove scalatest dependency
Update to Akka 2.5.3
Uptate to Akka HTTP 10.0.8

### CODE REVIEW

**Comment by jlprat** [comment]
I left the Scalatest dependency commented out, to make it easy for people using the seed with older versions of Akka HTTP

**Review by raboof — APPROVED** [review]
Not sure those commented-out dependencies are needed, but can't hurt

**Comment by jlprat** [comment]
I can also remove them, but if somebody wants to use version 10.0.7 it won't compile without those

File: src/main/g8/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies {
     compile 'com.typesafe.akka:akka-http_2.12:$akka_http_version$'
     compile 'com.typesafe.akka:akka-stream_2.12:$akka_version$'
     testCompile 'com.typesafe.akka:akka-http-testkit_2.12:$akka_http_version$'
-    testCompile 'org.scalatest:scalatest_2.12:3.0.1'
+//    testCompile 'org.scalatest:scalatest_2.12:3.0.1' // Uncomment this dependency when using Akka HTTP 10.0.7 or prior.
```
Comment (ktoso) [inline src/main/g8/build.gradle]: I'd rather remove it completely, we want to give "clean slate" and not polute new people's learning with some legacy thingy :)

**Comment by ktoso** [comment]
I vote for removing completely, LGTM after that ;)

**Comment by jlprat** [comment]
Done :)

**Comment by ktoso** [comment]
Thanks :)

**Pull request merged**